apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php

```diff
@@ -43,9 +43,29 @@ class WP_Template {
 	 * A8C_WP_Template constructor.
 	 */
 	public function __construct() {
-		$this->current_theme_name = get_option( 'stylesheet' );
```

Comment:
Just a note that we should probably refactor all usages of `get_option( 'stylesheet' )` and replace them with `get_stylesheet`. It's basically the same but applies the `stylesheet` filter on top of the result.